pkg/cleaner/jobcontroller.go

```diff
@@ -74,13 +73,13 @@ func NewCleanupJob(bd *v1alpha1.BlockDevice, volMode VolumeMode, namespace strin
 	mountName := "vol-mount"
 
 	if volMode == VolumeModeBlock {
-		input := "if=/dev/zero"
-		output := "of=" + bd.Spec.Path
-		blockSize := "bs=1M"
-		// get no of blocks required for a block size of 1M
-		blockCount := bd.Spec.Capacity.Storage / 1024 / 1024
-		count := "count=" + strconv.FormatUint(blockCount, 10)
-		jobContainer.Command = getCommand(BlockCleanerCommand, input, output, blockSize, count)
+		// wipefs erases the filesystem signature from the block
+		// -a    wipe all magic strings
+		// -f    force erasure
+		wipeCommand := BlockCleanerCommand + " -af " + bd.Spec.Path
+
+		jobContainer.Command = []string{"/bin/sh", "-c"}
+		jobContainer.Args = []string{wipeCommand}
```

Comment:
`jobContainer.Command = getCommand(BlockCleanerCommand, "-af", bd.Spec.Path)`
to make it more cleaner ?

Reply:
`getCommand` returns a slice of strings. We are passing a string to container args right.